parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -526,7 +529,7 @@ ensureKubelet() {
     fi
     chmod 0600 "${KUBELET_DEFAULT_FILE}"
 
-    if [ "${ENABLE_SECURE_TLS_BOOTSTRAPPING}" == "true" ] || [ -n "${TLS_BOOTSTRAP_TOKEN}" ]; then
+    if [ "${ENABLE_SECURE_TLS_BOOTSTRAPPING}" = "true" ] || [ -n "${TLS_BOOTSTRAP_TOKEN}" ]; then
```

Comment:
I removed this in a recent PR, you'll want to rebase

Reply:
no worries. Will do when I resolve the conflicts

Reply:
done